src/_cffi_src/openssl/ssl.py

```diff
@@ -688,6 +689,12 @@
 
 static const long Cryptography_HAS_SSL_CTX_CLEAR_OPTIONS = 1;
 
+#ifdef OPENSSL_NO_TLSEXT
```

Comment:
cryptography doesn't support OpenSSL compiled without TLS extensions so you can always assume this is not set. Do you still need it then?